### PR TITLE
add localisation:

### PR DESCRIPTION
**Pull request by wswebcreation**
This PR adds:
- German Language
- test for localization

### CODE REVIEW

File: MyDemoAppUITests/Tests/LocalizationTest.swift [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        XCTAssert(app.staticTexts["Sauce Labs Rucksack"].exists)
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        app.terminate()
+    }
+}
+
+import XCTest
```
Comment (vijaysharm) [inline MyDemoAppUITests/Tests/LocalizationTest.swift]: Is there a reason the import needs to be at the bottom of the file?

File: MyDemoAppUITests/Tests/LocalizationTest.swift [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        // 4. Launch the app
+        app.launch()
+
+        // Check the German text
+        XCTAssert(app.staticTexts["Sauce Labs Rucksack"].exists)
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        app.terminate()
```
Comment (vijaysharm) [inline MyDemoAppUITests/Tests/LocalizationTest.swift:47]: What happens if you call terminate without a launch call? does terminate throw an exception?

**Review by vijaysharm — APPROVED** [review]

File: MyDemoAppUITests/Tests/LocalizationTest.swift (hunk 1 above)
Comment (wswebcreation) [reply MyDemoAppUITests/Tests/LocalizationTest.swift]: ohh, Christ, will fix it

File: MyDemoAppUITests/Tests/LocalizationTest.swift (hunk 2 above)
Comment (wswebcreation) [reply MyDemoAppUITests/Tests/LocalizationTest.swift:47]: Great question, I checked it by commenting out the code and only have this

```swift
import XCTest

class LocalizationTest: XCTestCase {
    var app: XCUIApplication!

    override func setUp() {
        super.setUp()
        continueAfterFailure = true
        app = XCUIApplication()
    }

    func testLocalization() throws {
    }

    override func tearDown() {
        super.tearDown()
        app.terminate()
    }
}

```

this is the log

```log
2023-07-11 13:28:53.580786+0200 MyDemoAppUITests-Runner[87260:1145427] [SceneConfiguration] Info.plist contained no UIScene configuration dictionary (looking for configuration named "(no name)")
2023-07-11 13:28:53.581100+0200 MyDemoAppUITests-Runner[87260:1145427] [SceneConfiguration] Info.plist contained no UIScene configuration dictionary (looking for configuration named "(no name)")
2023-07-11 13:28:53.585745+0200 MyDemoAppUITests-Runner[87260:1145427] Running tests...
Test Suite 'Selected tests' started at 2023-07-11 13:28:54.004
Test Suite 'MyDemoAppUITests.xctest' started at 2023-07-11 13:28:54.005
Test Suite 'LocalizationTest' started at 2023-07-11 13:28:54.005
Test Case '-[MyDemoAppUITests.LocalizationTest testLocalization]' started.
    t =     0.00s Start Test at 2023-07-11 13:28:54.005
    t =     0.03s Set Up
    t =     0.03s Tear Down
    t =     0.03s     Terminate com.saucelabs.mydemoapp.ios:0
Test Case '-[MyDemoAppUITests.LocalizationTest testLocalization]' passed (0.048 seconds).
Test Suite 'LocalizationTest' passed at 2023-07-11 13:28:54.054.
	 Executed 1 test, with 0 failures (0 unexpected) in 0.048 (0.049) seconds
Test Suite 'MyDemoAppUITests.xctest' passed at 2023-07-11 13:28:54.054.
	 Executed 1 test, with 0 failures (0 unexpected) in 0.048 (0.049) seconds
Test Suite 'Selected tests' passed at 2023-07-11 13:28:54.054.
	 Executed 1 test, with 0 failures (0 unexpected) in 0.048 (0.050) seconds
```

So it works without an error 😅

**Pull request merged**